src/style/colors.ts

```diff
@@ -220,38 +220,19 @@ export class Colors {
     return this.shouldSupportDarkMode && Scheme.getSchemeType() === 'dark' ? _.reverse(palette) : palette;
   });
 
-  generateDesignTokens(color: string, mode: 'light' | 'dark' = 'light') {
-    return mode === 'light' ? this.generateLightModeTokens(color) : this.generateDarkModeTokens(color);
-  }
-
-  private generateLightModeTokens(color: string) {
-    const colorPalette: string[] = this.generatePalette(color);
-    const color30 = colorPalette[2];
-    const color50 = colorPalette[4];
-    const color70 = colorPalette[6];
-    const color80 = colorPalette[7];
-
-    const mainColor = this.isDark(color) ? color : color30;
-
-    return {
-      $backgroundPrimaryHeavy: mainColor,
-      $backgroundPrimaryLight: color80,
-      $backgroundPrimaryMedium: color70,
-      $iconPrimary: mainColor,
-      $iconPrimaryLight: color50,
-      $textPrimary: mainColor,
-      $outlinePrimary: mainColor
-    };
-  }
-
-  private generateDarkModeTokens(color: string) {
-    const colorPalette: string[] = _.reverse(this.generatePalette(color));
+  private generateDesignTokens({primaryColor, dark}: {primaryColor: string; dark?: boolean}) {
```

Comment:
Why the parameter here is an object? This method is private so no worries about changing its signature in the future. The parameters could be simply `primaryColor: string and dark: boolean`.

Reply:
Done